README.md

```diff
@@ -251,6 +252,25 @@ The parameters are as follows:
 	- Accepted values: Functions accepting content and returning content, or `none`.
 	- Default value: `none`.
 
+- `frontmatter`:
+  A function to define how markdown metadata affects the document.
+  Contents inside the frontmatter section (defined by `---\nfrontmatter\n---\n` at the top of the file) are hidden from the main body and can be accessed here.
+  The underlying format for the frontmatter is determined by the user and should be parsed accordingly.
+  For example,
+  ```typst
+  #cmarker.render(
+    read("input.md"),
+    frontmatter: (meta, body) => {
+      meta = yaml(bytes(meta))
+      set document(title: meta.at("title"))
+      body
+    },
+  )
+  ```
+  - Accepted values: Functions accepting (metadata, content) and returning content, or `none`.
```

Comment:
It occurred to me that it might be nice to have:
- `yaml_metadata: "frontmatter"` parse the YAML internally inside the `render` function, but also support
- `yaml_metadata: "frontmatter_raw"` to get the metadata as a string.